### DISCUSSION TITLE
Spotube installation problem on ios

### DISCUSSION THREAD

**Discussion by Raphkoos** [KRTirtho/spotube#1667] [Q&A]
Hi,
I installed Spotube correctly on my PC but I would also like to have it on my iPhone. I manage to download the ipa file, once it is downloaded I click on it but it only shows me the file. I would like help on how to download the application. 
Thanks for the help, kings regards.

![IMG_2312](https://github.com/KRTirtho/spotube/assets/167633657/b0411271-5a47-4967-9f36-d5e98ad80892)
![IMG_2311](https://github.com/KRTirtho/spotube/assets/167633657/ab76e08d-9f9d-483f-b9c9-38777fa8a3ef)

**Comment by tigerkiller55891** [comment]
Since spotube isn't on the official appstore you have to sideload it using something like sideloadly. Unfortunately since apple doesn't like people sideloading they made it difficult and it can't just be a few clicks like anything else.
